src/components/Toast.js

```diff
@@ -19,16 +19,17 @@ class Toast extends React.Component {
         style: PropTypes.oneOfType([
             PropTypes.object,
             PropTypes.bool
-        ])
+        ]),
+        stylesheet: PropTypes.object
```

Comment:
This can be of course `PropTypes.shape(...` to make it accommodate the shape being checked type checked 😄